src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -104,6 +108,11 @@ public function editAction(int $orderReturnId, Request $request): Response
         $formHandler = $this->get('prestashop.core.form.identifiable_object.handler.order_return_form_handler');
 
         try {
+            $editableOrderReturn = $this->getQueryBus()->handle(
```

Comment:
By doing this you duplicate the call to the `GetOrderReturnForEditing` which is already done by the form handler. This should be avoided, if you need to get some data from the order return you can use `$form->getData()` which will be populated with the data from the `GetOrderReturnForEditing` and you'll be able to get the `getOrderReturnStateId` data you need for rendering